backend/tests/export/test_init.py

```diff
@@ -147,6 +147,9 @@ def test_export_votes(db_session, tmp_path):
         ],
         responsible_committees=[Committee["AFET"]],
         result=VoteResult.ADOPTED,
+        amendment_subject=None,
+        amendment_number=None,
+        texts_adopted_reference="A10-123-123",
```

Comment:
Not relevant from a technical perspective as we don’t do anything with it, but I think this would be the expected format

```suggestion
        texts_adopted_reference="P10_TA(2025)0123",
```